### PR TITLE
[Web App] novu functionalilty to received push notification - added

### PR DESCRIPTION
**Pull request by vitto-moz**
https://www.notion.so/saladtech/App-Receive-Notifications-from-Novu-in-the-Web-App-0c78de7b514949e0ab42e11338f33668?pvs=4

### CODE REVIEW

File: packages/web-app/src/components/NovuProviderWrapper/NovuProviderWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      <NovuProvider
+        applicationIdentifier={config.novuAppId}
+        subscriberId={currentProfile?.id}
+        initialFetchingStrategy={{ fetchNotifications: true }}
+      >
+        {children}
+      </NovuProvider>
+    )
+  }
+  return children
```
Comment (Maks19) [inline packages/web-app/src/components/NovuProviderWrapper/NovuProviderWrapper.tsx:25]:
```suggestion
  return currentProfile?.id ? (
    <NovuProvider
      applicationIdentifier={config.novuAppId}
      subscriberId={currentProfile?.id}
      initialFetchingStrategy={{ fetchNotifications: true }}
    >
      {children}
    </NovuProvider>
  ) : (
    children
  )
```

Comment (vitto-moz) [reply packages/web-app/src/components/NovuProviderWrapper/NovuProviderWrapper.tsx:25]: why so ? :)
Does initial version look wrong?

File: packages/web-app/src/components/NovuProviderWrapper/NovuProviderWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { NovuProvider } from '@novu/notification-center'
+import { observer } from 'mobx-react'
+import type { ReactElement } from 'react'
+import { config } from '../../config'
+import { getStore } from '../../Store'
+
+interface Props {
```
Comment (rners01) [inline packages/web-app/src/components/NovuProviderWrapper/NovuProviderWrapper.tsx:7]:
```suggestion
interface NovuProviderWrapper Props {
```

File: packages/web-app/src/components/NovuProviderWrapper/NovuProviderWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+export const NovuProviderWrapper = observer(({ children }: Props): ReactElement | null => {
+  const store = getStore()
+  const { currentProfile } = store.profile
+  if (currentProfile?.id) {
+    return (
+      <NovuProvider
+        applicationIdentifier={config.novuAppId}
+        subscriberId={currentProfile?.id}
```
Comment (rners01) [inline packages/web-app/src/components/NovuProviderWrapper/NovuProviderWrapper.tsx:18]:
```suggestion
        subscriberId={currentProfile.id}
```

**Review by rners01 — APPROVED** [review]

**Pull request merged**